demo/app/main-view-model.ts

```diff
@@ -1,17 +1,17 @@
 import { Observable } from 'tns-core-modules/data/observable';
 import * as dialogs from 'tns-core-modules/ui/dialogs';
 import { isAndroid } from 'tns-core-modules/platform';
-
+import { device } from "tns-core-modules/platform";
 import * as Permissions from 'nativescript-permissions';
-import { getAccessToken, setupCallListener, setupPushListener, Twilio } from 'nativescript-twilio';
+import { getAccessToken, setupCallListener, setupPushListener, Twilio , unregisterPushNotifications} from 'nativescript-twilio';
 
 declare var android: any;
 
 export class HelloWorldModel extends Observable {
   public message: string;
   public senderPhoneNumber: string = '+34606039750';
   // public phoneNumber: string = '+639171137700';
-  public phoneNumber: string = '+34605264081';
+  public phoneNumber: string = '+14692900583';
```

Comment:
I know that it was a previous mistake made by myself but, do you mind clearing those fields?

Reply:
Sure thing manuel thanks. Ive removed the fields and placed a comment to explain the field values